src/main/java/frc/robot/subsystems/AimingSubsystem.java

```diff
@@ -137,8 +154,8 @@ private void updateMotorModes() {
             break;
     }
 
```

Comment:
We don't need a neutralModeValue here because we aren't using CTRE at all in this subsystem

Reply:
Fixed it and removed all unused imports.